nullaway/src/main/java/com/uber/nullaway/CodeAnnotationInfo.java

```diff
@@ -269,7 +270,14 @@ private boolean isAnnotatedTopLevelClass(Symbol.ClassSymbol classSymbol, Config
       // make sure it's not explicitly configured as unannotated
       return !shouldTreatAsUnannotated(classSymbol, config);
     }
-    return false;
+    // Check if it is NullMarked inside a Library Model when in JSpecify Mode
+    if (config.isJSpecifyMode()) {
```

Comment:
For this PR, I think it's good to only check for the `@NullMarked` override in JSpecify mode, to limit impact on existing code.  If/when we are able to fully import annotations from something like jspecify/jdk, we may need a different strategy.  E.g., we could treat the whole class as `@NullMarked`, but outside JSpecify mode treat checks involving type variables as if the code is unmarked?  Need to think carefully about this, but not for this PR.